src/components/CustomStatusBar/CustomStatusBar.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { StatusBar } from 'react-native';
+
+type Props = {
+	textColor: 'light-content' | 'dark-content';
+};
+
+export const CustomStatusBar = ({ textColor }: Props) => {
+	return <StatusBar barStyle={textColor} />;
+};
```

Comment:
I can't see the point in using CustomStatusBar. It looks like using straight StatusBar is easier

Reply:
same thought